test/components/SuperpowerTile.js

```diff
@@ -0,0 +1,21 @@
+describe('SuperpowerTile', () => {
+  let wrapper,
+      name;
+
+  beforeEach(() => {
+    wrapper = mount(
+      <SuperpowerTile
+        name: 'Flight'
```

Comment:
We want an `=` here instead of a `:`. Additionally, it looks like Codeship is only running ONE of your React tests - please run locally to see what's going on? I don't think it's running this one!